redis/Cargo.toml

```diff
@@ -38,26 +38,26 @@ combine = { version = "4.6", default-features = false, features = ["std"] }
 
 # Only needed for AIO
 bytes = { version = "1", optional = true }
-futures-util = { version = "0.3.15", default-features = false, optional = true }
```

Comment:
Is there a reason to do this? Offhand it seems to me like the patch versions specified in here are useful from a documentation perspective if nothing else. If we don't actually know for sure that the `major.minor.0` versions of these crates will actually compile with the code here, it seems better to not go back and indicate as such now.

Reply:
Doesn't this limit users of the crate? The dependency resolver has to use this specific version of the dependency, and if they have another crate pointing at a different patch version, the dependency is compiled twice?

Reply:
I don't think it's limiting, though. I think it ensures that only versions >= what's actually been tested are used, which should generally result in fewer old bugs.

Reply:
removed.